tests/test_hassos.py

```diff
@@ -0,0 +1,27 @@
+"""Test Home Assistant OS functionality."""
+
+from awesomeversion import AwesomeVersion
+import pytest
+
+from supervisor.coresys import CoreSys
+
+URL_TEST = "https://version.home-assistant.io/stable.json"
+
+# pylint: disable=protected-access
+
+
+@pytest.mark.asyncio
+async def test_ota_url_generic_x86_64_rename(coresys: CoreSys) -> None:
+    """Test download URL generated."""
+
+    coresys.hassos._board = "intel-nuc"
+    coresys.hassos._version = AwesomeVersion("5.13")
+    await coresys.updater.fetch_data()
```

Comment:
I know the updater tests also do this, but we should patch I/O in tests

Reply:
Agreed. Let's fix it once updater is fixed.